src/index.ts

```diff
@@ -28,7 +28,9 @@ import logger from "./log";
 // set initial features according to environment variables
 initializeFeatures();
 
-if (__ENVIRONMENT__.CURRENT_ENV === __ENVIRONMENT__.DEV as number) {
+if (typeof __RX_PLAYER_DEBUG_MODE__ === "boolean" && __RX_PLAYER_DEBUG_MODE__) {
```

Comment:
Note: this magic line only works because:

```ts
typeof UNDECLARED_VARIABLE
```
return `undefined`, even in strict mode, instead of throwing.

This may be ugly, but as some use our code in their server-side-rendering logic, we cannot just assume `window` is there, we could also use a more general global but I thought just relying on typeof quirks were simpler to read (as well as being fully functional)